joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

left_wheel_effort_controller:
  type: effort_controllers/JointEffortController
  joint: axle_to_left_wheel

right_wheel_effort_controller:
  type: effort_controllers/JointEffortController
  joint: axle_to_right_wheel

left_steer_position_controller:
  type: effort_controllers/JointPositionController
  joint: chassis_to_left_steering
  pid: {p: 10.0, i: 0.01, d: 0.1}

right_steer_position_controller:
  type: effort_controllers/JointPositionController
  joint: chassis_to_right_steering
  pid: {p: 10.0, i: 0.01, d: 0.1}
